### DISCUSSION TITLE
Can we have containers listen only on certain host IPs?

### DISCUSSION THREAD

**Discussion by Anaxagoras-bc** [nextcloud/all-in-one#3980] [Ideas] [answered]
So i'm running into an issue where talk wants to listen on port 3478, but i have another service listening on 3478, but my host has multiple ip's.  I bind published ports to specific ips.  often in the docker compose i'd just publish the port as $IP1:3478:3478 for service 1 (talk), and $IP2 for service 2 (other app using the same port).

Is there a way to do this? and if not request it?

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one?tab=readme-ov-file#how-to-adjust-the-talk-port
